src/components/StatsTable.js

```diff
@@ -0,0 +1,32 @@
+import React, { PureComponent } from 'react';
```

Comment:
This will be hard for you :D ... but please remove this file since its not used and there will be KslTable component that will do the same job.